recipe/meta.yaml

```
@@ -40,42 +40,44 @@ outputs:
       run:
         - python >=3.8
 
-        # dependencies matched to pip
+        # dependencies matched to setup.cfg
+        - app-model>=0.0.8,<0.3.0a
```

All the other packages have a space between the name and version constraint. I assume this isn't actually needed, but would be nice to have consistency.

```suggestion
        - app-model >=0.0.8,<0.3.0a
```

---

Ah, yes, indeed!